bitnami/harbor/templates/exporter/exporter-dpl.yaml

```
@@ -0,0 +1,184 @@
+      {{- end }}
+      {{- if .Values.exporter.priorityClassName }}
+      priorityClassName: {{ .Values.exporter.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.exporter.schedulerName }}
+      schedulerName: {{ .Values.exporter.schedulerName }}
+      {{- end }}
+      {{- if .Values.exporter.serviceAccountName }}
+      serviceAccountName: {{ .Values.exporter.serviceAccountName }}
+      {{- end }}
+      {{- if .Values.exporter.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.exporter.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.exporter.initContainers }}
+      initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: exporter
+          image: {{ include "harbor.exporter.image" . }}
+          imagePullPolicy: {{ .Values.exporter.image.pullPolicy | quote }}
+          {{- if .Values.exporter.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.exporter.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.exporter.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.exporter.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.exporter.args "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.exporter.image.debug | quote }}
+            - name: HARBOR_DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "harbor.core" . }}-envvars
+                  key: POSTGRESQL_PASSWORD
+            - name: HARBOR_ADMIN_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "harbor.core" . }}-envvars
+                  key: HARBOR_ADMIN_PASSWORD
+            {{- if .Values.exporter.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.exporter.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            - configMapRef:
+                name: "{{ include "harbor.exporter" . }}-envvars"
+          ports:
+            - containerPort: {{ .Values.exporter.containerPorts.metrics }}
+              name: {{ include "harbor.metricsPortName" . }}
```

Again, hardcode the name:
```suggestion
              name: metrics
```